src/components/pagination/pageControls/pageNavigator/bar/bar.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, ReactElement } from 'react';
+import classNames from 'classnames/bind';
+import { Tooltip } from '@/components';
```

Comment:
Need to be changed to more specific import
```suggestion
import { Tooltip } from '@components/tooltip';
```
That was the reason for the circular dependency (bar -> components -> pagination -> ... -> bar) and the error in storybook
![image](https://github.com/reportportal/ui-kit/assets/31861805/5a6b46de-01df-4ca2-b740-65cc92ad59ce)